1_processing/4_viz_compendia_pca.ipynb

```diff
@@ -882,6 +882,28 @@
   {
```

Comment:
<p>Should you add var explained to these axes too?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/core-accessory-interactome/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Thanks for the suggestion. I'll add it to the axes as well. It was only added to PAO1, since the PA14 plot isn't shown in the paper. But I think you're right, that its good to be consistent